extensions/amp-story/1.0/_locales/en.js

```diff
@@ -182,22 +182,27 @@ const strings = {
     string: 'Play video',
     description: 'Label for a button to play the video visible on the page.',
   },
-  [LocalizedStringId.AMP_STORY_QUIZ_ANSWER_CHOICE_A]: {
+  [LocalizedStringId.AMP_STORY_INTERACTIVE_RESULTS_SCORE]: {
+    string: 'SCORE:',
+    description:
+      'Label for the results component preceding the score in percentages, should have `:` at the end',
```

Comment:
I'm not sure it is actually the correct in every language to end in a colon.

Reply:
True, removed that part